google/cloud/bigtable/options.h

```diff
@@ -82,6 +82,9 @@ struct MinConnectionRefreshOption {
  * Maximum time in ms to refresh connections.
  *
  * The server will disconnect idle connections before this time.
+ *
+ * @note If this value is less than the value of `MinConnectionRefreshOption`,
```

Comment:
Don't you need a similar warnings for the other value?

Reply:
I don't think so. Either
1. the min option is unset -> we give the user a default value
2. the min option is set -> it is used.

the max value on the other hand may be set AND change if the min value is set higher than it.